core/services/src/main/java/org/exoplatform/services/cms/webdav/WebDavServiceImpl.java

```diff
@@ -294,6 +300,13 @@ public Response get(@PathParam("repoName") String repoName,
       }
       return Response.serverError().build();
     }
+    SettingValue<?> downloadDocumentSettingValue =
+                                                 settingService.get(org.exoplatform.commons.api.settings.data.Context.GLOBAL.id("downloadDocumentStatus"),
+                                                                    Scope.APPLICATION.id("downloadDocumentStatus"),
+                                                                    "exo:downloadDocumentStatus");

```

Comment:
It would be better to not use the implementation details of Transfer Rule API here, but define a common API JAR where to define the Service API layer in order to reuse it inside other implementation JARs

Reply:
@boubaker where i can define the common api jar ?  in ecms or commons module or gatein-portal beside the settingservice ?

Reply:
ok, I see the problem, there is no Service layer for Transfer Rules API, in addition, the feature is mean't to change options in UI only without restrictions in backend. Thus, let's fix the frontend only.

Reply:
done